src/browser/components/MainPage.jsx

```diff
@@ -422,7 +422,7 @@ export default class MainPage extends React.Component {
           ) : null}
         </Grid>
         <TransitionGroup>
-          { (this.props.displayUrlOnHover !== true || this.state.targetURL === '') ?
+          { (this.props.displayUrlOnHover !== true || this.state.targetURL === '' || this.state.targetURL.includes(this.props.internalUrl)) ?
```

Comment:
Can you use regex matching for uchat/uchat-staging/uchat-dev. like using test function e.g. /^(uchat.*).uberinternal.com$/.test(this.props.internalUrl)